tests/metrics/workload_time/cor_create_time.sh.in

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test measures the time cc-oci-runtime (cor) takes from creating
+#  a container and before starting it.
+
+set -e
+
+if [ $# -ne 1 ]; then
+	echo -e "Usage: $0 <times to run>" && exit 1
+fi
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../common/test.common"
+
+TEST_NAME="Cor Start Time"
+LOG_FILE="/run/opencontainer/containers/cc-oci-runtime.log"
+IMAGE='busybox'
+TIMES="$1"
+RUNTIME='cor'
+testResultFileNet="${RESULT_DIR}/cor-start-time-with-network"
+testResultFileNoNet="${RESULT_DIR}/cor-start-time-without-network"
+PACKAGE_NAME="@PACKAGE_NAME@"
+
+function get_time(){
+	egrep "${PACKAGE_NAME}:debug:called as: (/[^/ ]*)+/${PACKAGE_NAME} create" ${LOG_FILE} | \
```

Comment:
This assumes the runtime is running in debug mode. It used to do that by default but now doesn't (see commit 781ff32f7393894004bdf9554be61ffa0e31f279).

We probably don't want these tests creating `/etc/cc-oci-runtime/cc-oci-runtime.sh.cfg` directly, but it should be documented somewhere that the tests assume the runtime is running in debug mode.